### PR TITLE
docs for new qual model

### PR DESCRIPTION
**Pull request by davidbenjamin**
@ldgauthier A user just pointed out that there are no docs for the new qual model.  I thought I put them in the repo a long time ago, but nope.  Here they are.  It's nothing you haven't seen before.

### CODE REVIEW

File: docs/gatk_qual.tex [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+\email{davidben@broadinstitute.org}
+\affiliation{Broad Institute, 75 Ames Street, Cambridge, MA 02142}
+
+\date{\today}
+
+\maketitle
+
+\section{Introduction}\label{introduction}
+
+The variant quality model is used in both \code{HaplotypeCaller} and \code{GenotypeGVCFs}.  We begin, in \code{HaplotypeCaller}, after deciding which alleles to consider and calculating likelihoods of each possible genotype, or, in \code{GenotypeGVCFs}, after merging input GVCFs, harmonizing their allele representations, and deciding which alleles to keep. Let $\ell_{sg} \equiv P({\rm reads}_s | g)$ be the likelihood of genotype $g$ in sample $s$.  Then a simple model for the genotypes and observed reads is
```
Comment (ldgauthier) [inline docs/gatk_qual.tex]: Isn't this part of the decision process for which alleles to keep in GenotypeGVCFs? Calculate the probability of AC > 0 for each allele and compare that with stand-emit-conf?

File: docs/gatk_qual.tex [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
..._B = 0.25$ should match the empirical distribution of allele frequencies.  Note that the old model essentially assumed that every site had an alt allele with a constant allele frequency of 1/1000.  We can achieve this by setting $\vpi \sim {\rm Dir}(\valpha)$, where $\valpha$ is a vector with one component per allele, and $\valpha_a$ is a prior pseudocount for allele $a$.  We can set these pseudocounts separately for the ref allele, SNPs, and indels\footnote{In principle we could have a complicated model for prior pseudocounts depending on context.  Furthermore, if we have a big call set like gnomAD, the prior pseudocounts are simply the allele counts from that call set.} to obtain the desired mean and standard deviation of allele frequencies.  Unlike previous approaches which shoehorned multiallelic sites into  biallelic model, this method extends naturally to multiallelic sites.  Since $\valpha$ is not a random variable we can ignore the normalization constant of the Dirichlet, which depends only on $\valpha$, to obtain $P(\vpi | \valpha) \propto  \prod_a \pi_a^{\alpha_a}$.  This, combined with Equations \ref{likelihood} and \ref{indicator}, gives the joint posterior distribution
```
Comment (ldgauthier) [inline docs/gatk_qual.tex]: Maybe this should say for gnomAD the _best_ prior pseudocounts are simply the allele counts from that callset.  You code doesn't actually use different priors for different sized callsets right now, right?

File: docs/gatk_qual.tex (hunk 1 above)
Comment (davidbenjamin) [reply docs/gatk_qual.tex]: True.  Changed it.

File: docs/gatk_qual.tex (hunk 2 above)
Comment (davidbenjamin) [reply docs/gatk_qual.tex]: Done.

**Comment by davidbenjamin** [comment]
Did the two things.

**Pull request merged**